src/snaphu/_unwrap.py

```diff
@@ -198,12 +200,29 @@ def normalize_and_validate_tiling_params(
     return ntiles, tile_overlap, nproc
 
 
+def nan_to_zero(arr: ArrayLike) -> np.ndarray:
```

Comment:
any reason not to use [`nan_to_num`](https://numpy.org/doc/stable/reference/generated/numpy.nan_to_num.html) ?

Reply:
I decided against using `nan_to_num` because it also converts $\pm$ inf, which doesn't seem like it's necessarily what we'd want here.